### PR TITLE
Initial setup/create and implement classes

### PR DESCRIPTION
**Pull request by trace-kadenyi**
Create three classes:
 - Person
 - Student
 - Teacher

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    if of_age? || parent_permission
+      true
+    else
+      false
+    end
+  end
+  # declare private and public methods
+  private :of_age?
+  public :can_use_services?
+end
```
Comment (smunozmo) [inline person.rb]: - [x] You did a great job, congrats on it! 🎉 However, let's make it even better by using the Ruby best practices for syntax, usually when your Class has a private method you place it at the bottom like this:

```
class Person

  def initialize
      ...
  end

  def can_use_services? 
     ...
  end

  private 

  def of_age?
    ...
  end
end
```
With this syntax, you can remove lines 26 and 27 and that's it! 😉

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  end
+
+  # public method
+  def can_use_services?
+    if of_age? || parent_permission
+      true
+    else
+      false
+    end
+  end
```
Comment (smunozmo) [inline person.rb]: [OPTIONAL] This is correct, it works properly, just a friendly tip, you could also get the same result by saying:

```
def can_use_services?
    of_age? || parent_permission
end
```

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @TracK92, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Person, Student and Teacher classes are set up correctly, congrats on it! 🎉 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 2 above)
Comment (trace-kadenyi) [reply person.rb]: [smunozmo](https://github.com/smunozmo) thanks for the suggestions

**Review by gemmen29 — APPROVED** [review]
Hi @TracK92, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**